docs/troubleshooting/new-connection.md

```diff
@@ -24,11 +22,24 @@ One workaround is to manually pull the latest version of every connector you'll
 
 If the above workaround does not fix your problem, please report it [here](https://github.com/airbytehq/airbyte/issues/1462) or in our [Slack](https://slack.airbyte.io).
 
-### **Connection refused errors when connecting to a local db**
+### Connection refused errors when connecting to a local db
```

Comment:
same

Reply:
I changed the title to `##` level  for all files in troubleshooting section